msteams-platform/bots/bot-features.md

```diff
@@ -47,6 +48,105 @@ The [Bot Framework](https://dev.botframework.com/) is a rich SDK used to create
+
+> [!IMPORTANT]
+> Bot applications within Microsoft Teams are available in GCC-High through [Azure bot Service](/azure/bot-service/channel-connect-teams).
+
+> [!NOTE]
+> * Bots in GCCH only support  up to manifest version v1.10.
+> * Image URL's in Adaptive Cards are not supported in GCCH environment. You can replace an image URL with Base64 encoded DataUri.
+> * Bot channel registration in Azure Government will provision web app bot, app service (app service plan), and application insights also but it doesn't support to provision the azure bot service only (no app service).
+>   <details>
+>   <summary><b>If you want to do bot registration only</b></summary>
+>
+>   * Go to the resource group and manually delete the unused resources. Such as the app service, app service plan (if you created during bot registration), and the application insights (if you choose to enable it during bot registration).
+>   * You can also use az-cli to do bot registration:
+>
+>     1. Sign into azure and set the subscription <br> 
+>           &nbsp; az cloud set –name  "AzureUSGovernment" <br> 
+>           &nbsp; az account set –name "`subscriptionname/id`".<br>
+>     1. Create app registration  
+>           &nbsp; az ad app create --display-name "`name`" <br> 
+>           &nbsp; --password "`password`" --available-to-other-tenants.<br> 
+>           Your app id would be created here.<br>
+>     1. Create bot resource <br>
+>           &nbsp; az bot create –resource-group "`resource-group`"<br>
+>           &nbsp; --appid "`appid`"<br>
+>           &nbsp; --name "`botid`"<br>
+>           &nbsp; --kind "registration".<br>
+>
+> </details>
+
+For GCCH environment, you need to register a bot using [Azure Government portal](https://portal.azure.us).
+
+:::image type="content" source="../assets/videos/abs-bot.gif" alt-text="Azure Government portal":::
+<br>
+<br>
+The following changes are needed within the bot for GCC-High environment:
+<br>
+<br>
+<details>
+<summary><b>Configuration changes</b></summary>
+
+As the bot registration occurs in Azure Government portal, ensure to update the bot configurations to connect to Azure govermnet instances. Following are the configuration details:
+
+| Configuration Name | Value |
+|----|----|
+| ChannelService | `https://botframework.azure.us` |
+| OAuthUrl | `https://tokengcch.botframework.azure.us` |
+| ToChannelFromBotLoginUrl | `https://login.microsoftonline.us/MicrosoftServices.onmicrosoft.us` |
+| ToChannelFromBotOAuthScope | `https://api.botframework.us` |
+| ToBotFromChannelTokenIssuer | `https://api.botframework.us`  |
+| BotOpenIdMetadata | `https://login.botframework.azure.us/v1/.well-known/openidconfiguration` |
+
+</details>
+<br>
+<details>
+<summary><b>Update to appsettings.json & startup.cs</b></summary>
+
+1. **Update appsettings.json:**
+
+    * Set `ConnectionName` to the name of the OAuth connection setting you added to your bot.
```

Comment:
Let's swap these two bullet points. All bots have an app id and password, but not all bots have an OAuth connection setting.

Reply:
Actually, wait a sec... this specific section (appsettings.json) don't seem specific to GCCH at all. Dont' all bots do this?

Reply:
So, this was done as a buffer while Jim updated his documents (why we have the note for the correct link) 

Once Jim publishes, he documents we will remove this section.